pkg/generate/app/templatelookup.go

```diff
@@ -57,9 +61,13 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
 						Score:       score,
 						Template:    template,
 					})
+					break
 				}
 			}
 		}
+		if exact {
+			break
```

Comment:
I don't see any logic that's even using StopOnMatch in this PR.  what am i missing?